README.md

```diff
@@ -1,2 +1,121 @@
-# aws-nextflow-workflows
-AWS CloudFormation templates for deploying Nextflow infrastructure
+# Nextflow Workflows Infrastructure
+
+The AWS infrastructure for hosting a private instance of [Nextflow Tower](https://tower.nf/) and executing [Nextflow workflows](https://nextflow.io/) is defined in this repository and deployed using [CloudFormation](https://aws.amazon.com/cloudformation/) via [Sceptre](https://sceptre.cloudreach.com/).
+
+## Onboarding
+
+To complete these onboarding instructions, you will need a project name (_e.g._ `imcore`, `amp-ad`, `commonmind`), and a stack name, _i.e._ the project name with the suffix `-project` (_e.g._ `imcore-project`, `amp-ad-project`, `commonmind-project`).
+
+**N.B.** Anytime that `<stack_name>` apears with the angle brackets, replace this placeholder with the actual stack name without the angle brackets.
+
+1. Install [AWS CLI v2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) and [Docker Desktop](https://www.docker.com/products/docker-desktop).
```

Comment:
This is under the assumption people are working on mac or windows right?

Reply:
I updated the Docker link to also include installation instructions for Linux.